server/src/service/PkiService.js

```diff
@@ -298,6 +298,39 @@ exports.splitChainIntermediateCertificate = async (body) => {
   return chainInfo;
 };
 
+/**
+ *
+ */
+exports.retrieveFirstAndRemainingIntermediateChainCerts = async (intermediateChain) => {
+  const beginCertRegex = /(?=-----BEGIN)/g;
+
+  let chainInfo = [];
+  let remainingIntermediateChainInfo = '';
+  let count = ((intermediateChain && intermediateChain.match(/BEGIN/g)) || []).length;
+  // split the intermediatesChain into a list of certInfo
+  if (count > 0) {
+    let intermediateChains = intermediateChain.split(beginCertRegex);
+    for (let index = 0; index < intermediateChains.length; index++) {
+      let chain = intermediateChains[index];
```

Comment:
and this should be called intermediateCert

Reply:
Changed to intermediateCert per comment.